srcpkgs/network-manager-applet/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'network-manager-applet'
 pkgname=network-manager-applet
 version=1.18.0
-revision=1
+revision=2
 build_style=meson
 configure_args="-Dselinux=false"
 hostmakedepends="dbus-glib-devel glib-devel intltool pkg-config"
 makedepends="ModemManager-devel NetworkManager-devel iso-codes jansson-devel
  libappindicator-devel libapparmor-devel libdbusmenu-gtk3-devel
  libgnome-keyring-devel libgudev-devel libnotify-devel libnma-devel
  libsecret-devel polkit-devel mobile-broadband-provider-info"
-depends="NetworkManager hicolor-icon-theme iso-codes"
+depends="NetworkManager hicolor-icon-theme iso-codes libnma"
```

Comment:
This will be detected automatically.
`SONAME: libnma.so.0 <-> libnma>=1.8.30_1`

The dangling entry of `libnm-gtk` in `common/shlibs` was causing xbps to incorrectly detect `libnm-gtk` as runtime dependency, with that gone explicitly listing it in depends should not be necessary.

Reply:
Gotcha. So `network-manager-applet` doesn't need to be revbumped, then?

Reply:
Nevermind, ericonr just explained to me why it does need to be revbumped.